app/Repositories/ContactRepositoryCacheFilesystem.php

```diff
@@ -0,0 +1,38 @@
+<?php namespace App\Repositories;
+
+use App\Models\Contact;
+use Cache;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
+
+class ContactRepositoryCacheFilesystem implements ContactRepository
```

Comment:
A very simple cache that uses the Laravel Cache facade (which uses the filesystem in our config).